src/views/HomeView.vue

```diff
@@ -1,35 +1,99 @@
 <script setup lang="ts">
-import VerticalBox from '../components/VerticalBox.vue'
-import SimpleLink from '../components/SimpleLink.vue'
-import PortfolioSection from '../components/portfolio_elements/PortfolioSection.vue'
-import PortfolioPreview from '../components/portfolio_elements/PortfolioPreview.vue'
-
-import img_UnmaskedPreview from '@/assets/portfolio/games/unmasked/UnmaskedPreview.png'
-import img_HushPreview from '@/assets/portfolio/games/hush/HushPreview.png'
-import img_MouthPreview from '@/assets/portfolio/games/giantwalkthroughmouth/MouthPreview.png'
-import img_DentaLabPreview from '@/assets/portfolio/games/dentalab/DentaLabPreview.png'
+import VerticalBox from '@components/VerticalBox.vue'
+import HorizontalBox from '@components/HorizontalBox.vue'
+import SimpleLink from '@components/SimpleLink.vue'
+import PortfolioSection from '@components/portfolio_elements/PortfolioSection.vue'
+import PortfolioPreview from '@components/portfolio_elements/PortfolioPreview.vue'
 </script>
 
 <template>
-  <VerticalBox>
-    <h1 class="center-wrapper">Work.</h1>
-    <HorizontalBox class="center-wrapper">
-      <SimpleLink target="Work">Games</SimpleLink>
-      <SimpleLink target="VR Research">VR Research</SimpleLink>
-      <SimpleLink target="Software">Software</SimpleLink>
-    </HorizontalBox>
-
-    <h2 class="left-wrapper">Games.</h2>
-    <PortfolioSection>
-      <PortfolioPreview :imageSrc="img_UnmaskedPreview" />
-      <PortfolioPreview :imageSrc="img_HushPreview" />
-    </PortfolioSection>
+  <main>
+    <VerticalBox>
+      <h1 class="center-wrapper">Work.</h1>
+      <HorizontalBox class="center-wrapper">
+        <div v-for="(category, index) in categories" :key="index" class="category">
+          <SimpleLink :href="'#' + category.name">{{ category.name }}</SimpleLink>
```

Comment:
Was there a reason to use this kind of concatenation here?

Reply:
Habit lol